projects/WTI-API/packageWTI.xml

```diff
@@ -59,28 +65,38 @@
 		<echo message="Staged project files at ${staging.dist.dir}"/>
 	</target>
 
-	<!-- set the value of ${repo.version} --> 
-    <target name="gitversion">
-		<available classname="org.eclipse.jdt.core.JDTCompilerAdapter" property="build.compiler" value="org.eclipse.jdt.core.JDTCompilerAdapter" />
-    	
-    	<!-- set ${git-hash} to either to the value of GIT_COMMIT if it exists in the environment, or HEAD if it doesn't -->
-         <condition property="git_hash" value="${env.GIT_COMMIT}" else="HEAD">
-            <isset property="env.GIT_COMMIT"/>
-        </condition>
-    	
-    	<!-- call git to obtain the number (count) of the most recent version associated with the value of ${git-hash} -->
+    <target name="-execgit" unless="exists.GITHUB_RUN_NUMBER">
```

Comment:
And again... comments explaining why the 10 lines of XML code are being replaced by the one line would be helpful...